go/actions-client.go

```diff
@@ -67,10 +67,10 @@ type ActionsClient struct {
 }
 
 // Create creates a new action.
-func (a ActionsClient) Create(ctx context.Context, req *CreateActionRequest) (*actions.CreateActionResponse, error) {
+func (a ActionsClient) Create(ctx context.Context, req *CreateActionRequest) (*actions.CreateActionResponse, *SdkAPIError) {
 	callProperties, err := a.callPropertiesCreator.GetTeamsLevelCallProperties(ctx)
 	if err != nil {
-		return nil, err
+		return nil, NewSdkAPIError(err, CreateActionRPC, actionsFeatureGroupID)
```

Comment:
This could be misleading for anyone seeing these errors IMO, as it didn't fail in the request itself.